src/lib/orionTypes/OrionValueType.cpp

```diff
@@ -43,6 +43,7 @@ const char* valueTypeName(const orion::ValueType _type)
   case ValueTypeBoolean:      return "Boolean";
   case ValueTypeObject:       return "Object";
   case ValueTypeVector:       return "Vector";
+  case ValueTypeNull:         return "Null";
```

Comment:
Good! This will be useful in the future at mongoBackend to rightly persist this kind of values.

NTC